docs/reference/edot-cloud-forwarder/aws.md

```diff
@@ -0,0 +1,384 @@
+---
+navigation_title: AWS
+description: Set up the EDOT Cloud Forwarder for AWS to bring your AWS logs to Elastic Observability.
+applies_to:
+  serverless:
+    observability: preview
+products:
+  - id: cloud-serverless
+  - id: observability
+  #- id: edot-cf
+---
+
+# EDOT Cloud Forwarder for AWS
+
+{{edot-cf}} for AWS provides the EDOT Collector as a Lambda function that collects and forwards logs to Elastic Observability on {{serverless-full}}. {{edot-cf}} for AWS supports the following log sources:
+
+| Log source | Description |
+| --- | --- |
+| VPC Flow | Logs generated by a VPC (Virtual Private Cloud). |
+| ELB Access | Logs generated by an ELB (Elastic Load Balancer). |
+| CloudWatch | Logs generated by AWS CloudWatch. |
+
+Read on to learn how to set up {{edot-cf}} for AWS.
+
+::::{important}
+{{edot-cf}} for AWS is not supported on {{ech}} or self-managed {{stack}}.
+::::
+
+## Prerequisites
+
+To collect logs using {{edot-cf}} for AWS you need:
+
+::::{tab-set}
+
+:::{tab-item} VPC Flow
+
+To collect VPC Flow logs, you need:
+
+- A VPC (Virtual Private Cloud).
+- An S3 bucket for storing flow logs.
+- A flow log configured with the S3 bucket as the destination.
+
+:::
+
+:::{tab-item} ELB Access
+
+To collect ELB (Elastic Load Balancer) Access logs, you need:
+
+- An ELB of any type (ALB, NLB, CLB).
+- An S3 bucket to store the access logs.
+- Access logging enabled, with the bucket as the destination.
+
+:::
+
+:::{tab-item} CloudWatch
+
+To collect CloudWatch logs, you need:
+
+- A CloudWatch Log Group where logs are stored.
+
+:::
+::::
+
+In addition, you need to know the URL of the managed OTLP endpoint and the API key for authentication.
+
+:::{include} ../_snippets/serverless-endpoint-api.md
+:::
+
+In the CloudFormation templates, the OTLP endpoint is set as `OTLPEndpoint`, and the API key is set as `ElasticAPIKey`.
+
+## Deployment considerations
+
+Before deploying {{edot-cf}} for AWS, keep these points in mind:
+
+- Deploy a separate CloudFormation stack for each log type, for example VPC Flow Logs, ELB Logs, or CloudWatch Logs. Each CloudFormation stack can only process one log source and format at a time.
+- Logs stored in S3 must be placed in separate buckets. Each log type should reside in its own dedicated bucket.
+
+## Download the template [download-templates]
+
+Download the CloudFormation template to deploy the appropriate stack based on your log source:
+
+| Log Source | CloudFormation template |
+| --- | ------------------------------------------------ |
+| S3 logs | `s3://edot-cloud-forwarder/v0/latest/cloudformation/s3_logs-cloudformation.yaml` |
+| CloudWatch logs | `s3://edot-cloud-forwarder/v0/latest/cloudformation/cloudwatch_logs-cloudformation.yaml` |
+
```

Comment:
These URLs might change based on this discussion - https://github.com/elastic/edot-cloud-forwarder-aws/pull/132#discussion_r2172652565

In short, having `https://` is helpful to use with AWS Console. `s3://` is only supported by CLI. But CLI support both of them :)